File: 프로그래머스/unrated/181929. 원소들의 곱과 합/원소들의 곱과 합.js
function solution(num_list) {
    let sum = 0
    let multi = 1

    let result_sum_2 = num_list.map(item => sum += item).at(-1) ** 2
    let result_multi = num_list.map(item => multi *= item).at(-1)

   return result_sum_2 > result_multi ? 1 : 0
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
  public static int solution(int[] num_list) {
    int sum = 0;
    int multiple = 1;
    // for문
    for (int i = 0; i < num_list.length; i++) {
      // 더하기
      sum += num_list[i];
      // 곱하기
      multiple *= num_list[i];
    }

    // 삼항 연산자
    return multiple < sum * sum ? 1 : 0;
  }
}